### PR TITLE
✨ run-analysis action files

### PR DESCRIPTION
**Pull request by laurentsimon**
Files to run the scorecard action.
See this file to see how to use it from a workflow https://github.com/laurentsimon/scorecard-action-test-4/blob/main/.github/workflows/scorecard-analysis.yml#L24
The branch is set to this PR branch for testing only - by default main branch is used and it not necessary.

### CODE REVIEW

File: run-analysis/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+fi
+
+# It's important to change directories here, to ensure
+# the files in SARIF start at the source of the repo.
+# This allows GitHub to highlight the file.
+cd "$GITHUB_WORKSPACE"
+
+if [[ "$GITHUB_EVENT_NAME" == "pull_request"* ]]
+then
+    $SCORECARD_BIN --local . --format sarif --show-details --policy="$SCORECARD_POLICY_FILE" > "$SCORECARD_SARIF_FILE"
```
Comment (asraa) [inline run-analysis/entrypoint.sh:46]: just curious, will this depend on the repo being checked out? (i.e. use actions/checkout somewhere) -- maybe this is already done?

**Review by asraa — APPROVED** [review]
Thanks! LGTM

Assuming eventually you'd like a README in this repo (esp to define the policy, since there's no default one) -- happy to start a PR for this unless you have some copy-paste-able doc already ready!

File: run-analysis/entrypoint.sh (hunk 1 above)
Comment (laurentsimon) [reply run-analysis/entrypoint.sh:46]: absolutely https://github.com/ossf/scorecard/blob/main/.github/workflows/scorecard-analysis.yml#L22

Comment (asraa) [reply run-analysis/entrypoint.sh:46]: thanks!

**Review by asraa — APPROVED** [review]

**Comment by naveensrinivasan** [comment]
## Scorecard Results</br>
</br>

**Pull request merged**